### DISCUSSION TITLE
need SendAndWait function

### DISCUSSION THREAD

**Discussion by laomms** [jchristn/SuperSimpleTcp#54] [Q&A]
great job !
Is it possible add SendAndWait to client class ?

**Comment by jchristn** [comment]
Not easily.  SimpleTcp doesn't provide any means of framing, meaning there is no metadata attached to the data you're sending over the wire, so there's no way to determine if the response received is unsolicited or from a client's SendAndWait request.  It's relatively easy in WatsonTcp (which does have framing), but not in SimpleTcp.
